src/main/java/org/folio/consortia/service/impl/SharingSettingServiceImpl.java

```diff
@@ -149,6 +168,40 @@ private UUID publishRequest(UUID consortiumId, PublicationRequest publicationReq
     return null;
   }
 
+  @Scheduled(fixedRate = 10) // Poll every 10 ms
```

Comment:
This method will be executed always when app is running, we can not use apporach here with this sceduled annotation

Reply:
fixed